WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefsWrapper.kt

```diff
@@ -85,6 +85,10 @@ class AppPrefsWrapper @Inject constructor() {
         get() = AppPrefs.shouldScheduleCreateSiteNotification()
         set(shouldSchedule) = AppPrefs.setShouldScheduleCreateSiteNotification(shouldSchedule)
 
+    var isJetpackMigrationFlowCompleted: Boolean
+        get() = AppPrefs.isJetpackMigrationFlowCompleted()
+        set(isCompleted) = AppPrefs.setJetpackMigrationFlowCompleted(isCompleted)
+
```

Comment:
This looks ok for now, given the "WIP" state of the data layer at the moment, but just mentioning that this responsibility will eventually belong to the migration orchestrator, and it's likely that we will not use a pref flag for this, since we will need to check that the db is populated anyway before proceeding. This is partially touched upon in this issue: https://github.com/wordpress-mobile/WordPress-Android/issues/17406. After migration succeeds, the migration will not be attempted again.

If this is necessary for this PR, I think it's ok to leave it in, but if not, it's probably best to remove it, do avoid duplicating the responsibilities. Wdyt?

Reply:
> This looks ok for now, given the "WIP" state of the data layer at the moment, but just mentioning that this responsibility will eventually belong to the migration orchestrator, and it's likely that we will not use a pref flag for this, since we will need to check that the db is populated anyway before proceeding. This is partially touched upon in this issue: #17406. After migration succeeds, the migration will not be attempted again.
> 
> If this is necessary for this PR, I think it's ok to leave it in, but if not, it's probably best to remove it, do avoid duplicating the responsibilities. Wdyt?

Thank you for your comment on this @mkevins 🙇 
I added it to provide way to avoid the disable flow when the feature flag is enabled, but don't see any other reason for keeping and my expectation was as you mentioned, that it might not be needed at all.

Since we're behind a temporary feature flag to test the migration flow I guess it's ok to accept the temporary behavior that will keep showing the migration flow every time the app is started when logged in.

Reply:
Addressed in commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17425/commits/a82026bfc80bfe9da0773c59966daa10c705a223